docs/feature_i2c_eeprom.md

```diff
@@ -0,0 +1,20 @@
+# I2C EEPROM Driver
```

Comment:
I almost feel like this should go in the "for makers and modders" section, rather than features.  

Given what it is, and how it works .... 

Also, don't forget to add a line to _summary.md, or it won't appear in the sidebar.